src/main.rs

```diff
@@ -29,8 +28,17 @@ struct Args {
 
   #[arg(short = 'l', help = "Linear readback (show explicit dups)")]
   pub linear: bool,
-  #[arg(short = 'O', default_value = "1", help = "Optimization level (0 or 1)", value_parser = opt_level_parser)]
-  pub opt_level: OptimizationLevel,
+
+  #[arg(
+    short = 'O',
+    value_delimiter = ' ',
+    action = clap::ArgAction::Append,
+    long_help = r#"Enables or disables the given optimizations
```

Comment:
Isn't there a field to specify possible allowed values for this option? I think there was something like that, but maybe not if your option is type String

Reply:
With the derive I don't think it is possible